src/util/Logger.ts

```diff
@@ -23,4 +27,17 @@ export function log(title: string, message: string, type?: 'log' | 'warn' | 'err
     }
 
     if (str) Webhook(str)
+
+    if (cluster.isMaster) {
+        // 如果是主进程，不设置定时器，直接返回
```

Comment:
why chinese?

Reply:
The comment means: If it's the main process, also known as the watchdog process, it is responsible for monitoring the log output of other processes to decide whether to restart them. Therefore, the main process might not output logs for a long time, so there's no need to set a timer for the main process.

> Haha, this was an oversight on my part. The original code from the repository had many issues when used in China, so I made quite a few modifications. One day, I noticed an issue where someone requested an auto-restart feature, so I took a portion of my code and pushed it. However, I didn't realize that I missed this part. My original code was all commented in Chinese, and I specifically changed it to English before pushing.